server/apps/publish/archive_publish.py

```diff
@@ -933,6 +934,11 @@ class CorrectPublishService(BasePublishService):
 
     def on_update(self, updates, original):
         updates[ITEM_OPERATION] = ITEM_CORRECT
+        if updates.get('renditions', []) and original.get(ITEM_TYPE) == CONTENT_TYPE.PICTURE:
+            crop_service = get_resource_service('archive_crop')
+            for key in updates.get('renditions', {}).keys():
```

Comment:
Invoking `keys()` is not really needed here, since the default dictionary iterator already iterates over its keys.